services/_system.md

```diff
@@ -85,6 +85,19 @@ Period between packets of data when streaming in milliseconds.
 
 Read-only value of the sensor, also reported in streaming.
 
+    const min_reading: i32 @ 0x104
+
+The lowest value that can be reported by the sensor.
+
+    const max_reading: i32 @ 0x105
+
+The highest value that can be reported by the sensor.
+
+    ro reading_error: i32 @ 0x106
```

Comment:
sensors typically use accuracy, eg. % of full scale value. should we use this instead?

Reply:
This page is an interresting read: https://www.ni.com/en-us/innovations/white-papers/13/sensor-terminology.html#:~:text=Back%20to%20top-,Accuracy,scale%20or%20in%20absolute%20terms.

Reply:
At least for temp sensors, it's absolute (0.2C), though it's bigger at the end of the range. Similar for humidity. I guess % is equivalent but absolute values are more clear-defined.
<img width="737" alt="Screen Shot 2021-01-13 at 2 23 22 PM" src="https://user-images.githubusercontent.com/10673976/104457966-ff3e3400-55aa-11eb-8721-a46cf2216134.png">

Reply:
maybe we can change this register to return a sequence of points and we do a linear interpoluation of the error.

Reply:
It should give you the current error. So in the example above, if it reads -20C it should return 0.5C error, and if it reads 25C it should return 0.2C error. The curves can have any shape I think.